docs/how-clerk-works/overview.mdx

```diff
@@ -304,6 +304,8 @@ However, server-rendered applications present a unique challenge. Server-to-serv
 1. FAPI receives the request with the `__client` cookie
 1. FAPI validates the authentication state and issues a new session token
 
+<Video src="/docs/images/how-clerk-works/handshake.mp4" width="1600" height="900" autoPlay muted loop playsInline />
```

Comment:
It won't actually make a difference because the _ratio_ is correct, but `width` and `height` should be set to the video's natural size:

```suggestion
<Video src="/docs/images/how-clerk-works/handshake.mp4" width="1920" height="1080" autoPlay muted loop playsInline />
```